src/Containers/MovieDetailsContainer/MovieDetailsContainer.module.css

```diff
@@ -0,0 +1,13 @@
+.headerLink {
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin-top: 10px;
+  margin-right: 10px;
```

Comment:
These styles are common between "Search" button and "Add movie" button.
They can be extracted to a separate class.